raiden/transfer/channel.py

```diff
@@ -2378,33 +2359,18 @@ def handle_channel_settled(
 
 
 def handle_channel_deposit(
-    channel_state: NettingChannelState,
-    state_change: ContractReceiveChannelDeposit,
-    block_number: BlockNumber,
+    channel_state: NettingChannelState, state_change: ContractReceiveChannelDeposit
 ) -> TransitionResult[NettingChannelState]:
-    deposit_transaction = state_change.deposit_transaction
-
-    if is_transaction_confirmed(deposit_transaction.deposit_block_number, block_number):
```

Comment:
AFAICT `is_transaction_confirmed` can be removed as well

Reply:
Well spotted! I've noticed that before, but must have missed it during the actual code removal.